packages/core/helper-plugin/src/components/DateTimePicker.tsx

```diff
@@ -1,10 +1,14 @@
-import React, { useEffect } from 'react';
+import * as React from 'react';
 
-import { DateTimePicker } from '@strapi/design-system';
+import { DateTimePicker, DateTimePickerProps } from '@strapi/design-system';
 
 // TODO: remove DateTimePicker component from the helper-plugin in V5
-const DateTimePickerLegacy = (props) => {
-  useEffect(() => {
+
+/**
+ * @deprecated Use the DateTimePicker from the Design System instead.
+ */
+const DateTimePickerLegacy = (props: DateTimePickerProps) => {
+  React.useEffect(() => {
     if (process.env.NODE_ENV === 'development') {
```

Comment:
While we're here, can we change this code block so we're just doing the `once` method please? We should not need to use an effect nor check if the ENV is dev :)